r/tests/testthat/test-dplyr-funcs-datetime.R

```diff
@@ -1130,6 +1130,17 @@ test_that("difftime works correctly", {
     test_df_with_tz
   )
 
+  test_df_with_tz %>%
+    arrow_table() %>%
+    mutate(
+      secs2 = difftime(
+        as.POSIXct("2022-03-07", tz = "Europe/Bucharest"),
+        time1,
+        units = "secs"
+      )
+    ) %>%
+    collect()
```

Comment:
Is there an expectation for this test? Also, is the timezone here something important about Bucharest or intended to be not-UTC default?

Reply:
Removed. This whole chunk is probably an oversight after a rebase. `tz` swapped for `"Pacific/Marquesas"`.